jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/FileSystemPool.java

```diff
@@ -43,6 +43,15 @@
     private static final Logger LOG = LoggerFactory.getLogger(FileSystemPool.class);
     public static final FileSystemPool INSTANCE = new FileSystemPool();
 
+    private static final Map<String, String> ENV_MULTIRELEASE_RUNTIME;
+
+    static
+    {
+        Map<String, String> env = new HashMap<>();
+        env.put("releaseVersion", "runtime");
```

Comment:
Don't forget to add here a comment with the link to the doc covering this env parameter.